src/inspector/view/qml/MuseScore/Inspector/notation/lines/SlurAndTieSettings.qml

```diff
@@ -81,8 +81,8 @@ Column {
 
         model: [
             { text: qsTrc("inspector", "Auto"), value: SlurTieTypes.TIE_PLACEMENT_AUTO },
-            { text: qsTrc("inspector", "Inside"), value: SlurTieTypes.TIE_PLACEMENT_INSIDE },
-            { text: qsTrc("inspector", "Outside"), value: SlurTieTypes.TIE_PLACEMENT_OUTSIDE }
+            { iconCode: IconCode.TIE_INSIDE, value: SlurTieTypes.TIE_PLACEMENT_INSIDE },
+            { iconCode: IconCode.TIE_OUTSIDE, value: SlurTieTypes.TIE_PLACEMENT_OUTSIDE }
```

Comment:
No accessibility text?

Reply:
Accessibility text can be added by setting `title`, i.e. for example
```
{ iconCode: IconCode.TIE_INSIDE, value: SlurTieTypes.TIE_PLACEMENT_INSIDE, title: qsTrc("inspector", "Inside") },
```